### PR TITLE
Supplies for Software Carpentry

### PR DESCRIPTION
**Pull request by mbonsma**
I bought more sticky notes, a whiteboard marker, and some sharpies for nametags.

### CODE REVIEW

File: treasurer/reimbursing/submitted/2017-01-31-madeleine-bonsma.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+# Reimbursement form
+
+- Purpose of expense:
+- Name:
```
Comment (lwjohnst86) [inline treasurer/reimbursing/submitted/2017-01-31-madeleine-bonsma.md]: Please include the purpose and your name :smile:

File: treasurer/reimbursing/submitted/2017-01-31-madeleine-bonsma.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Receipt(s) list
+
+<!-- Replace the below example, following the format provided, and adding items as needed -->
```
Comment (lwjohnst86) [inline treasurer/reimbursing/submitted/2017-01-31-madeleine-bonsma.md]: remove

File: treasurer/reimbursing/submitted/2017-01-31-madeleine-bonsma.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+<!-- Example -->
```
Comment (lwjohnst86) [inline treasurer/reimbursing/submitted/2017-01-31-madeleine-bonsma.md]: remove

**Comment by lwjohnst86** [comment]
@mbonsma you mind fixing up those changes before I merge in?

**Comment by mbonsma** [comment]
Sure thing, thanks for the reminder @lwjohnst86.

**Pull request merged**